echopype/process/process_deprecated.py

```diff
@@ -310,8 +310,8 @@ def get_MVBS(self, source_postfix='_Sv', source_path=None,
         ping_size = MVBS_ping_size if MVBS_ping_size is not None else self.MVBS_ping_size
 
         # Range must have ping_time or it will error. Range does not have ping time for AZFP when temps are averaged
```

Comment:
"Range (echo_range) must have ping_time or it will error. Range does not have ping_time for AZFP when temps are averaged"